lib/node_modules/@stdlib/stats/base/ndarray/stdev/lib/main.js

```diff
@@ -0,0 +1,73 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var numelDimension = require( '@stdlib/ndarray/base/numel-dimension' );
+var getStride = require( '@stdlib/ndarray/base/stride' );
+var getOffset = require( '@stdlib/ndarray/base/offset' );
+var getData = require( '@stdlib/ndarray/base/data-buffer' );
+var ndarraylike2scalar = require( '@stdlib/ndarray/base/ndarraylike2scalar' );
+var strided = require( '@stdlib/stats/strided/stdev' ).ndarray;
+
+
+// MAIN //
+
+/**
+* Computes the standard deviation of a one-dimensional ndarray.
+*
+* @param {ArrayLikeObject<Object>} arrays - array-like object containing ndarrays
+* @returns {number} standard deviation
+*
+* @example
+* var Float64Array = require( '@stdlib/array/float64' );
+* var scalar2ndarray = require( '@stdlib/ndarray/from-scalar' );
+* var ndarray = require( '@stdlib/ndarray/base/ctor' );
+*
+* var opts = {
+*     'dtype': 'float64'
+* };
+*
+* var xbuf = new Float64Array( [ 2.0, 1.0, 2.0, -2.0, -2.0, 2.0, 3.0, 4.0 ] );
+* var x = new ndarray( opts.dtype, xbuf, [ 4 ], [ 2 ], 1, 'row-major' );
+*
+* var correction = scalar2ndarray( 1.0, opts );
+*
+* var v = stdev( [ x, correction ] );
+* // returns 2.5
+*/
+function stdev( arrays ) {
+	var correction;
+	var stride;
+	var x;
+	var N;
+
+	x = arrays[ 0 ];
+	correction = ndarraylike2scalar( arrays[ 1 ] );
+
+	N = numelDimension( x, 0 );
+	stride = getStride( x, 0 );
+	return strided( N, correction, getData( x ), stride, getOffset( x ) );
```

Comment:
```suggestion
	var x;

	x = arrays[ 0 ];
	correction = ndarraylike2scalar( arrays[ 1 ] );

	return strided( numelDimension( x, 0 ), correction, getData( x ), getStride( x, 0 ), getOffset( x ) ); // eslint-disable-line max-len
```

Consistency with elsewhere.